test-files/MessageContext.dynamic.tsx

```
@@ -0,0 +1,6 @@
+// This is a mock dynamic file
+export const useMessage = () => {
+  return {
+    message: 'Hello World'
```

ESLint

[error] Insert `,`

(prettier/prettier)